src/js/components/Pagination/Pagination.js

```diff
@@ -33,13 +34,14 @@ const Pagination = forwardRef(
       page: pageProp,
       size,
       step: stepProp,
+      stepSelector,
```

Comment:
The index.d.ts and propTypes files need update to reflect the `stepSelector` prop

Reply:
left off cause not sure we'll go with `stepSelector` but will add todo for after our conversation on a clear direction

Reply:
Sounds good, that makes sense